src/ip/QosConfig.cc

```diff
@@ -533,7 +533,11 @@ Ip::Qos::setSockTos(const int fd, tos_t tos, int type)
     // unless we pass it a 32-bit variable storing 8-bits of data.
     // NP: it is documented as 'int' for all systems, even those like Linux which accept 8-bit char
     //     so we convert to a int before setting.
+#if _SQUID_MINGW_
```

Comment:
> This change is independant of the changes in PR #1341

This PR can certainly be viewed as independent, but if I do that, then I would have to block this PR because `_SQUID_MINGW_` semantics on the official code is not yet compatible with this PR (even though the code may _compile_ -- the compiler does not understand macro semantics). It will become compatible after PR 1341 is merged. I prefer to just wait for that PR to land, so that we can move steadily forward in one direction.

Same for the other similar recent PRs, of course.

Reply:
I do not understand what you mean. PR #1343 does not change _SQUID_MINGW_ in any way. The macro exists today in `master` branch and is set for exactly the build environment which this new code is needed.

Reply:
You probably meant "PR 1341". Please see https://github.com/squid-cache/squid/pull/1341#discussion_r1199839754, where the discussion about those PR 1341 changes is (or should be) happening.

Reply:
> Alex: This PR can certainly be viewed as independent, but if I do that, then I would have to block this PR because `_SQUID_MINGW_` semantics on the official code is not yet compatible with this PR (even though the code may _compile_ -- the compiler does not understand macro semantics).

I found a way to let this PR merge independent from PR 1341: Squid already has a bit of code that uses `_SQUID_MINGW_` outside `_SQUID_WINDOWS_` sections. We can merge this new code before PR 1341 lands because this PR uses the same (old) approach (and it does not conflict with PR 1341 changes). I am removing my negative vote from this and similar recent PRs and will re-review them accordingly.